spec/services/one_off_redirect_fix_service_spec.rb

```diff
@@ -10,4 +10,11 @@
     described_class.update_redirects(edition_1)
     expect(edition_1.reload.redirects.first[:destination]).to eq edition_4.base_path
   end
+
+  it "raises an error if caught in a redirect loop" do
```

Comment:
Has a redirect loop happened in your tests?

I think we'll need to avoid raising an exception, as we wouldn't want the rest of the job to fail. Probably best just to skip over that edition and log the details.